network/p2p/throttled_handler.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package p2p
+
+import (
+	"context"
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+)
+
+var _ Handler = (*ThrottledHandler)(nil)
+
+type ThrottledHandler struct {
+	Handler
+	Throttler Throttler
+}
+
+func (t ThrottledHandler) AppGossip(ctx context.Context, nodeID ids.NodeID, gossipBytes []byte) error {
+	if err := t.Throttler.Throttle(ctx, nodeID, 1); err != nil {
```

Comment:
why do we hard code token here should be a const regardless

Reply:
Ah sorry i need to re-base this PR on the new throttler. The new throttler doesn't expose this anymore.